validator/lib/validateTileset.js

```diff
@@ -35,6 +35,18 @@ function validateTopLevel(tileset) {
     if (!defined(tileset.geometricError)) {
         return 'Tileset must declare its geometricError as a top-level property.';
     }
+
+    if (!defined(tileset.asset)){
```

Comment:
Another style suggestion that I missed before, also add a space before the curly brace for these:

`if (!defined(tileset.asset)) {`